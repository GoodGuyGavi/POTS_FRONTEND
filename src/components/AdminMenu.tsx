import React from "react";
import { Card, ListGroup, Nav } from "react-bootstrap";
import MenuList from "./MenuItem";
import AppTitle from "./AppTitle";

export interface IAdminMenuProps {
  title?: any;
  desc?: any;
  menuItem?: any;
}

const AdminMenu = (props: IAdminMenuProps) => {
  const { title, desc, menuItem } = props;

  return (
    <Card style={{ width: "18rem" }}>
      <AppTitle title={title} desc={desc} />
      <ListGroup>
        <Nav.Item>
          <Nav.Link href="">
            <MenuList menuItem={menuItem} />
          </Nav.Link>
        </Nav.Item>
      </ListGroup>
    </Card>
  );
};

export default AdminMenu;
