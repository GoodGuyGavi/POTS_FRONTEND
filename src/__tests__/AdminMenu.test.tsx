/**
 * @format
 */

import React from "react";
import { shallow } from "enzyme";
import AdminMenu from "../components/AdminMenu";

const props = {
  title: "Title",
  desc: "Purchase Order Tracking System",
  menuItem: [
    {
      id: 1,
      menuOption: "Dashboard"
    },
    {
      id: 2,
      menuOption: "Orders"
    },
    {
      id: 3,
      menuOption: "Supplier"
    }
  ]
};

describe("Admin Menu", () => {
  test("renders correctly", () => {
    expect(shallow(<AdminMenu {...props} />)).toMatchSnapshot();
  });
});
