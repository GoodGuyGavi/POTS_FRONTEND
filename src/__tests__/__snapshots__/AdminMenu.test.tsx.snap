// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin Menu renders correctly 1`] = `ShallowWrapper {}`;
